front/app/containers/UsersEditPage/index.tsx

```diff
@@ -35,48 +32,40 @@ const Container = styled.main`
   overflow-x: hidden;
 `;
 
-// To have two forms with an equal width,
-// the forms need to be wrapped with a div.
-// https://stackoverflow.com/questions/34993826/flexbox-column-direction-same-width
-const Wrapper = styled.div``;
-
-export default () => {
-  const { data: appConfig } = useAppConfiguration();
+const UsersEditPage = () => {
   const passwordLoginActive = useFeatureFlag({ name: 'password_login' });
   const { data: authUser } = useAuthUser();
-  const loaded = appConfig !== undefined && authUser !== undefined;
```

Comment:
After removing appConfig, we only checked whether authUser is not strictly undefined, which seems to be always true.